domain/build.gradle

```
@@ -0,0 +1,18 @@
+plugins {
+    id 'java-library'
+    id 'org.jetbrains.kotlin.jvm'
+}
```

순수 코틀린 모듈로 만들어주시면 좋을 것 같습니다. 😘
```suggestion
plugins {
    id 'kotlin'
}
```